src/sugar3/graphics/notebook.py

```diff
@@ -15,21 +15,23 @@
 # Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 # Boston, MA 02111-1307, USA.
 
-"""Notebook class
+'''
+Notebook class
 
-This class create a Gtk.Notebook() widget supporting
-a close button in every tab when the 'can-close-tabs' gproperty
-is enabled (True)
-
-STABLE.
-"""
+This class creates a Gtk.Notebook() widget supporting
+a close button in every tab when the `can-close-tabs` gproperty
+is enabled (True).
+'''
 
 from gi.repository import Gtk
 from gi.repository import GObject
 
 
 class Notebook(Gtk.Notebook):
-
+    '''
+    Notebook class that creates a Gtk.Notebook. It is possible to set
```

Comment:
How do you set the can close tabs prop from the contructor?  Maybe give an example like (untested):

    Notebook(can_close_tabs=false)

I think that is what the kwargs are doing